src/main/java/com/imaee/propinq/buildings/controllers/implementations/BuildingController.java

```diff
@@ -0,0 +1,27 @@
+package com.imaee.propinq.buildings.controllers.implementations;
+
+import com.imaee.propinq.buildings.controllers.interfaces.IBuildingController;
+import com.imaee.propinq.buildings.controllers.responses.BuildingResponse;
+import com.imaee.propinq.buildings.services.interfaces.IBuildingService;
+
+import java.util.List;
+import java.util.UUID;
+
+public class BuildingController implements IBuildingController {
+
+    private final IBuildingService buildingService;
+    public BuildingController(IBuildingService buildingService) {
+        this.buildingService = buildingService;
+    }
+    @Override
+    public List<BuildingResponse> getBuildings() {
+
+        return buildingService.getBuildings();
+    }
```

Comment:
Borremos esta línea. Orden visual. La implementación arranca en la línea siguiente de la apertura del método
```suggestion
   public List<BuildingResponse> getBuildings() {
        return buildingService.getBuildings();
    }
```